NDTensors/src/tupletools.jl

```diff
@@ -180,13 +180,13 @@ end
 
 Remove the value at pos and insert the elements in val
 """
-function insertat(t::NTuple{N}, val::NTuple{M}, pos::Integer) where {N,M}
+function insertat(t::Tuple, val::Tuple, pos::Integer)
+  N, M = length(t), length(val)
+  @boundscheck checkbounds(Base.OneTo(length(t)), pos)
```

Comment:
```suggestion
  @boundscheck checkbounds(Base.OneTo(N), pos)
```
since `length(t)` was already extracted.